src/connectors/refinement-list/__tests__/connectRefinementList-test.js

```diff
@@ -111,11 +110,11 @@ describe('connectRefinementList', () => {
     });
 
     // test that rendering has been called during init with isFirstRendering = true
-    expect(rendering.callCount).toBe(1);
+    expect(rendering.mock.calls.length).toBe(1);
     // test if isFirstRendering is true during init
-    expect(rendering.lastCall.args[1]).toBe(true);
+    expect(rendering.mock.calls[0][1]).toBe(true);
```

Comment:
You could use the shorthand method: `.toHaveBeenLastCalledWith(expect.any(Object), true)`